src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/30.diff.sql

```diff
@@ -0,0 +1,41 @@
+SET XACT_ABORT ON
```

Comment:
Do we need to do an `IF EXISTS (DELETE/ALTER) INDEX` on the problematic indexes?

Reply:
Yes, handled by DROP_EXISTING=ON

Reply:
`DROP_EXISTING=ON` allows for online updates